docs/book/advanced-guide/getting-started-on-aws.md

```diff
@@ -0,0 +1,335 @@
+# Getting started with AWS
+
+This step-by-step guide explains how to set up and configure all the infrastructure necessary to run a ZenML pipeline on AWS.
+
+## Prerequisites
+
+- [Docker](https://www.docker.com/) installed and running.
+- [kubectl](https://kubernetes.io/docs/tasks/tools/) installed.
+- The [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) installed and authenticated.
+- ZenML and the integrations for this tutorial stack installed:
+    ```shell
+    pip install zenml
+    zenml integration install aws s3 kubernetes
+    ```
+
+## Setting up the AWS resources
+
+All the AWS setup steps can either be done using the AWS UI or CLI. Simply select the tab for your preferred option and let's get started.
+First open up a terminal which we'll use to store some values along the way which we'll need to configure our ZenML stack later.
+
+### Artifact Store (S3 bucket)
+
+{% tabs %}
+{% tab title="AWS UI" %}
+
+- Go to the [S3 website](https://s3.console.aws.amazon.com/s3/buckets).
+- Click on `Create bucket`.
+- Select a descriptive name and a region. Let's also store these values in our terminal:
+    ```shell
+    REGION=<REGION> # for example us-west-1
+    S3_BUCKET_NAME=<S3_BUCKET_NAME>
+    ```
+
+{% tab title="AWS CLI" %}
+
+```shell
+# Replace the <PLACEHOLDERS> with a name for your bucket and the AWS region for your resources
+# Select one of the region codes for <REGION>: https://docs.aws.amazon.com/general/latest/gr/rande.html#regional-endpoints
+REGION=<REGION>  
+S3_BUCKET_NAME=<S3_BUCKET_NAME>
+
+aws s3api create-bucket --bucket=$S3_BUCKET_NAME \
+    --region=$REGION \
+    --create-bucket-configuration=LocationConstraint=$REGION
+```
+
+{% endtab %}
+{% endtabs %}
```

Comment:
I might be wrong but these represent two sets of opening and closing brackets, as such it should be:'

{% tabs %}
{% tab title="AWS UI" %}
UI STUFF
{% endtab %}
{% tab title="AWS CLI" %}
CLI STUFF
{% endtab %}
{% endtabs %}


```suggestion
{% tabs %}
{% tab title="AWS UI" %}

- Go to the [S3 website](https://s3.console.aws.amazon.com/s3/buckets).
- Click on `Create bucket`.
- Select a descriptive name and a region. Let's also store these values in our terminal:
    ```shell
    REGION=<REGION> # for example us-west-1
    S3_BUCKET_NAME=<S3_BUCKET_NAME>
    ```
{% endtab %}

{% tab title="AWS CLI" %}

```shell
# Replace the <PLACEHOLDERS> with a name for your bucket and the AWS region for your resources
# Select one of the region codes for <REGION>: https://docs.aws.amazon.com/general/latest/gr/rande.html#regional-endpoints
REGION=<REGION>  
S3_BUCKET_NAME=<S3_BUCKET_NAME>

aws s3api create-bucket --bucket=$S3_BUCKET_NAME \
    --region=$REGION \
    --create-bucket-configuration=LocationConstraint=$REGION
```

{% endtab %}
{% endtabs %}
```